api/services/vimeo.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * Vimeo API request to retrieve MP4 version for the requested video webpage 
+ */
```

Comment:
@joefhall Added description - wasn't sure how much detail they usually have but let me know if this is too vague!